src/services/usersService.js

```diff
@@ -0,0 +1,41 @@
+let users = [];
+
+const init = () => {
+  users = [];
+};
+
+const getAll = () => users;
+
+const getById = (id) => users.find((user) => user.id === Number(id)) || null;
+
+const create = (name) => {
+  const user = {
+    id: users.length,
```

Comment:
Create function for Id generation, it should be unique in array, because now this value can not be unique